connector/src/main/java/org/geysermc/connector/skin/SkinProvider.java

```diff
@@ -412,15 +412,35 @@ private static Skin supplyEars(Skin existingSkin, String earsUrl) {
 
         // if the requested image is a cape
         if (provider != null) {
+            // Scale capes based on size
+            BufferedImage newImage;
             if (image.getWidth() > 64) {
-                image = scale(image, 64, 32);
+                newImage = new BufferedImage(128, 64, BufferedImage.TYPE_INT_ARGB);
+                Graphics g = newImage.createGraphics();
+                g.drawImage(image, 0, 0, image.getWidth(), image.getHeight(), null);
+                g.dispose();
+                newImage = scale(newImage, 64, 32);
+            } else {
+                newImage = new BufferedImage(64, 32, BufferedImage.TYPE_INT_ARGB);
```

Comment:
Is this also for removing alpha? If so, add a comment?

Reply:
It shouldn't nor does it need to.

Reply:
I'm not sure why we need this else statement, then. I tested a normal-scale cape before this PR and it was fine.